common/app/experiments/Experiments.scala

```diff
@@ -7,6 +7,8 @@ import java.time.LocalDate
 object ActiveExperiments extends ExperimentsDefinition {
   override val allExperiments: Set[Experiment] =
     Set(
+      DCRJSBundleVariant,
```

Comment:
Where's this chap coming from?

Reply:
Whoopsies, can be moved to a separate PR.